### PR TITLE
[IA-667] Add 2 new Zendesk categories

### PR DESCRIPTION
**Pull request by mdegrego**

### CODE REVIEW

File: assistanceTools/zendesk.json [hunk 1]
```diff
@@ -92,6 +92,20 @@
...
+      },
+      {
+        "value": "suggerimento",
+        "description": {
+          "it-IT": "Suggerimento",
+          "en-EN": "Suggestion"
+        }
+      },
+      {
+        "value": "nessuno_dei_precedenti",
```
Comment (pietro909) [inline assistanceTools/zendesk.json:104]: 🤓 **suggestion**:
`nessuno_dei_precedenti` implies that the reader knows what the preceding categories are. I suggest using something more generic as could be `generic` itself or `uncategorized`.

Comment (mdegrego) [reply assistanceTools/zendesk.json:104]: @pietro909 These categories are reported by app IO in the same order as the JSON array.
Also, we have to implement this requirement: (see last two points)
https://pagopa.atlassian.net/browse/IA-667
Thanks, Marco

Comment (pietro909) [reply assistanceTools/zendesk.json:104]: Looking at the task you mentioned, I see only 2 points out of 4 addressed in this PR. 
Is this what we want?

Comment (mdegrego) [reply assistanceTools/zendesk.json:104]: @pietro909 Correct. The remaining points (Accorpare...) will be addressed later.

**Review by pietro909 — APPROVED** [review]

**Pull request merged**